sklearn/kernel_ridge/krc.py

```diff
@@ -0,0 +1,144 @@
+# License: BSD 3 clause
+
+from .base_ridge import _BaseKernelRidge
+from ..base import ClassifierMixin
+
+
+class KernelRidgeClassifier(ClassifierMixin, _BaseKernelRidge):
+    """Kernel Ridge Classifier.
+
+    Kernel Ridge Classifier (KRC) combines ridge
+    regression (linear least squares with l2-norm regularization), special
+    encoding for target and the kernel trick. It thus
+    learns a linear function in the space induced by the respective kernel and
+    the data. For non-linear kernels, this corresponds to a non-linear
+    function in the original space.
+
+    The form of the model learned by KRC is identical to support vector
+    classification (SVC). However, different loss functions are used:
+    KRR uses squared error loss while support vector classification uses
+    epsilon-insensitive loss, both combined with l2 regularization.
+    In contrast to SVC, fitting a multiclass KRC model can be done
+    at once, without using one-against-one or one-against-rest techniques.
+
+    Read more in the :ref:`User Guide <kernel_ridge>`.
+
+    Parameters
+    ----------
+    alpha : {float, array-like}, shape = [n_targets]
+        Small positive values of alpha improve the conditioning of the problem
+        and reduce the variance of the estimates.  Alpha corresponds to
+        ``(2*C)^-1`` in other linear models such as LogisticRegression or
+        LinearSVC. If an array is passed, penalties are assumed to be specific
+        to the targets. Hence they must correspond in number.
+
+    kernel : string or callable, default="rbf"
+        Kernel mapping used internally. A callable should accept two arguments
+        and the keyword arguments passed to this object as kernel_params, and
+        should return a floating point number.
+
+    gamma : float, default=None
+        Gamma parameter for the RBF, laplacian, polynomial, exponential chi2
+        and sigmoid kernels. Interpretation of the default value is left to
+        the kernel; see the documentation for sklearn.metrics.pairwise.
+        Ignored by other kernels.
+
+    degree : float, default=3
+        Degree of the polynomial kernel. Ignored by other kernels.
+
+    coef0 : float, default=1
+        Zero coefficient for polynomial and sigmoid kernels.
+        Ignored by other kernels.
+
+    kernel_params : mapping of string to any, optional
+        Additional parameters (keyword arguments) for kernel function passed
+        as callable object.
+
+    Attributes
+    ----------
+    dual_coef_ : array, shape = [n_samples] or [n_samples, n_targets]
+        Representation of weight vector(s) in kernel space
+
+    X_fit_ : {array-like, sparse matrix}, shape = [n_samples, n_features]
+        Training data, which is also required for prediction
+
+    classes_ : array of shape = [n_classes] or a list of such arrays
+        The classes labels (single output problem),
+        or a list of arrays of class labels (multi-output problem).
```

Comment:
I don't think we support multi-output classification here

Reply:
You are right, I'll remove it